File: .archon/workflows/sdlc/verify-runtime/scripts/gate-verified.py
"""Publish only a completed attempt's typed verdict."""

import json
import os

verdict = os.environ["INPUTS_STATUS"]
if verdict not in ("verified", "failed", "inconclusive"):
    raise ValueError(f"unexpected terminal assessment: {verdict}")
print(json.dumps({
    "verified": verdict == "verified",
    "verdict": verdict,
    "candidate": os.environ["INPUTS_CANDIDATE"],
    "checkout": os.environ["INPUTS_CHECKOUT"],
    "summary": os.environ["INPUTS_REASON"],
}))

File: .archon/workflows/sdlc/verify-runtime/scripts/gate-verified.ts
const result: unknown = JSON.parse(process.env.INPUTS_RESULT ?? 'null');
if (result === null || typeof result !== 'object' || !('status' in result) ||
    !['verified', 'failed', 'inconclusive'].includes(String(result.status)) ||
    !('candidate' in result) || !('checkout' in result) || !('reason' in result) || !('evidence' in result)) {
  throw new Error('unexpected terminal runtime assessment');
}
console.log(JSON.stringify({ verified: result.status === 'verified', verdict: result.status,
  candidate: result.candidate, checkout: result.checkout, summary: result.reason, evidence: result.evidence }));
